doc/sphinx-guides/source/developers/making-releases.rst

```diff
@@ -228,7 +273,12 @@ Create a new branch (any name is fine but ``prepare-next-iteration`` is suggeste
 
 Now create a pull request and merge it.
 
-For more background, see :ref:`base-supported-image-tags`.
+For more background, see :ref:`base-supported-image-tags`. For an example, see https://github.com/IQSS/dataverse/pull/10896
+
+Deploy Final Release on Demo
+----------------------------
```

Comment:
I moved the final deploy to demo down, after the release, because I added a "deploy release candidate to demo" step pretty early in the process.